src/sentry/models/notificationsetting.py

```diff
@@ -72,6 +72,7 @@ def provider_str(self) -> str:
             (NotificationSettingTypes.DEPLOY, "deploy"),
             (NotificationSettingTypes.ISSUE_ALERTS, "issue"),
             (NotificationSettingTypes.WORKFLOW, "workflow"),
+            (NotificationSettingTypes.ACTIVE_RELEASE, "activeRelease"),
```

Comment:
This was missing. This doesn't seem to require a migration even though its a modification of a Model. Include it anyways.